src/PostgREST/Config.hs

```diff
@@ -222,12 +222,21 @@ readAppConfig dbSettings optPath prevDbUri roleSettings roleIsolationLvl = do
     Left err ->
       return . Left $ "Error in config " <> err
     Right parsedConfig ->
-      Right <$> decodeLoadFiles parsedConfig
+      case processForbiddenValues parsedConfig of
+        Left err' ->
+          return . Left $ "Error in config " <> err'
```

Comment:
How about adding an io test for this? Maybe like:

https://github.com/PostgREST/postgrest/blob/86a48b5b4152e4a5a92e60128cca4eb50d1230d2/test/io/test_io.py#L1235-L1245

Reply:
thanks for the suggestion, I'm debugging the test now :)